src/stories/Input.stories.js

```diff
@@ -62,3 +63,14 @@ export const inputLength = () => ({
     </div>
   `
 })
+
+export const labelInput = () => ({
+  components: { LabelInput },
+  template: `
+    <div>
+      <label-input placeholder="Input here" label="Label Input" labelPosition="top"></label-input>
+      <label-input placeholder="Input here" label="Required Input" labelPosition="left" required></label-input>
```

Comment:
Can we please have them all with label on top? That is the template we are trying to go for

Reply:
![Screenshot 2020-05-03 at 8 03 20 AM](https://user-images.githubusercontent.com/15964566/80896886-eccde400-8d14-11ea-86e9-398f7a3c3777.png)
The component has option to choose between left and top position. Do you want me to remove the left option availability?

Reply:
ah you are right. No lets leave it open to choose the position of the label :)